src/søknad/forside/Forsideinformasjon.tsx

```diff
@@ -143,10 +117,9 @@ const Forsideinformasjon: React.FC<InnholdProps> = ({
             )}
             onChange={() => settBekreftelse(!harBekreftet)}
           >
-            <BlockContent
-              blocks={disclaimer}
-              serializers={{ types: { block: BlockRenderer } }}
-            />
+            <div className={'navds-body-short navds-body-long'}>
```

Comment:
er det ikke rart å bruke både "body-short" og "body-long" ? 🤔

Reply:
👍 Vi (jeg og C) tok en titt, vi gikk for å bruke BodyShort då det gav en bedre spacing i teksten.